src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -227,50 +242,56 @@ public static AbstractBuilding create(Colony colony, TileEntityColonyBuilding pa
             {
                 BlockPos loc = parent.getPosition();
                 Constructor<?> constructor = oclass.getDeclaredConstructor(Colony.class, BlockPos.class);
-                building = (AbstractBuilding)constructor.newInstance(colony, loc);
+                building = (AbstractBuilding) constructor.newInstance(colony, loc);
             }
             else
             {
-                Log.logger.error(String.format("TileEntity %s does not have an associated Building.", parent.getClass().getName()));
+                Log.logger.error(String
+                        .format("TileEntity %s does not have an associated Building.", parent.getClass().getName()));
             }
         }
-        catch (NoSuchMethodException | InstantiationException | InvocationTargetException | IllegalAccessException exception)
+        catch (NoSuchMethodException | InstantiationException | InvocationTargetException
+                | IllegalAccessException exception)
         {
-            Log.logger.error(String.format("Unknown Building type '%s' or missing constructor of proper format.", parent.getClass().getName()), exception);
+            Log.logger.error(String.format("Unknown Building type '%s' or missing constructor of proper format.", parent
+                    .getClass().getName()), exception);
         }
 
         return building;
     }
 
     /**
-     * Load data from NBT compound.
-     * Writes to {@link #buildingLevel}, {@link #rotation} and {@link #style}
+     * Load data from NBT compound. Writes to {@link #buildingLevel},
+     * {@link #rotation} and {@link #style}
      *
-     * @param compound      {@link net.minecraft.nbt.NBTTagCompound} to read data from
+     * @param compound
+     *            {@link net.minecraft.nbt.NBTTagCompound} to read data from
      */
     public void readFromNBT(NBTTagCompound compound)
     {
         buildingLevel = compound.getInteger(TAG_BUILDING_LEVEL);
 
         rotation = compound.getInteger(TAG_ROTATION);
         style = compound.getString(TAG_STYLE);
-        if("".equals(style))
+        if ("".equals(style))
         {
             Log.logger.warn("Loaded empty style, setting to classic");
             style = "classic";
         }
 
-        if(MaterialSystem.isEnabled)
+        if (MaterialSystem.isEnabled)
         {
             materialStore.readFromNBT(compound);
         }
     }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)